training_model/auth_utils.py

```
@@ -0,0 +1,68 @@
+    _load_environment_if_needed(cfg)
+
+    if cfg.other.hf_login:
+        hf_token = os.getenv("HF_TOKEN")
+        if hf_token is None:
+            raise OSError("Environment variable 'HF_TOKEN' is not set.")
+        login(token=hf_token)
+
+    # Weights & Biases login
+    wb_token = os.getenv("WANB_API")
+    if wb_token is None:
+        raise OSError("Environment variable 'WANB_API' is not set.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)